documentation/sdk/sveltekit.md

```diff
@@ -16,6 +16,28 @@ description: i18n SDK designed and fully integrated for SvelteKit.
 
 inlang provides an SDK that integrates seamlessly with SvelteKit.
 
+## Requirements
```

Comment:
I would not call this "Requirements" and certainly not put this as the first section in the docs ^^

Reply:
I just would make a "Further reading" section where we shortly mention where the files are located per default and that it can be customized :)

Reply:
I put it to the bottom and changed the name to `Further reading`